### PR TITLE
pk sprint1 task2

### LINKED ISSUE #14: [Team 1 Task 2]: Implement Azure Function for /users/{user_id} API
**Is your feature request related to a problem? Please describe.**
Project requires Azure Function to call the /users/{user_id} API

**Describe the solution you'd like**
A functional API for /users/{user_id}

**Describe alternatives you've considered**
N/A - feature is required as-is to function with rest of team/class

### PR DESCRIPTION
**Pull request by philckim**
fairly confident the logic is sound and that it would connect to the db. running shows functions as implemented. but cannot debug issues around pyodbc package on local machine...

### CODE REVIEW

**Review by RealDogDad — DISMISSED** [review]
Please add your .vscode/ dir to your .gitignore and request reviews again.

File: .gitignore
Comment (jlsnow301) [inline .gitignore]: Just click the lil commit button 
```suggestion
.venv
.vscode/
```

**Review by jlsnow301 — CHANGES_REQUESTED** [review]

**Review by All-Schuck-Up — COMMENTED** [review]
Looks good with recommended changes to gitignore

**Review by jlsnow301 — APPROVED** [review]
should be okay once merge conflicts are resolved

**Pull request merged**
